src/app/component/shared/components/calendar-base/calendar-base.component.ts

```diff
@@ -213,39 +237,167 @@ export class CalendarBaseComponent implements OnInit, OnDestroy {
 
   public formatDate(isMonthCalendar: boolean, dayItem) {
     if (isMonthCalendar) {
-      return `${dayItem.year}-${ dayItem.month + 1 < 10 ?
+      return `${dayItem.year}-${dayItem.month + 1 < 10 ?
         '0' + (dayItem.month + 1) : dayItem.month + 1}-${dayItem.numberOfDate < 10 ?
-        '0' + dayItem.numberOfDate : dayItem.numberOfDate}`;
+          '0' + dayItem.numberOfDate : dayItem.numberOfDate}`;
     } else {
-      return `${dayItem.date.getFullYear()}-${ dayItem.date.getMonth() + 1 < 10 ?
+      return `${dayItem.date.getFullYear()}-${dayItem.date.getMonth() + 1 < 10 ?
         '0' + (dayItem.date.getMonth() + 1) : dayItem.date.getMonth() + 1}-${dayItem.date.getDate() < 10 ?
-        '0' + dayItem.date.getDate() : dayItem.date.getDate()}`;
+          '0' + dayItem.date.getDate() : dayItem.date.getDate()}`;
+    }
+  }
+
+  formatSelectedDate(isMonthCalendar, dayItem: CalendarInterface) {
+    if (isMonthCalendar) {
+      return this.months[dayItem.month] + ' ' + dayItem.numberOfDate + ', ' + dayItem.year;
```

Comment:
use string interpolation instead

Reply:
https://dmitripavlutin.com/string-interpolation-in-javascript/

Reply:
fixed. thanks for the example.